llvm/lib/Target/RISCV/RISCVInstrInfo.td

```diff
@@ -1463,9 +1463,9 @@ def PROBED_STACKALLOC_RVV : Pseudo<(outs GPR:$sp),
                                []>,
                                Sched<[]>;
 let usesCustomInserter = 1 in
-def PROBED_STACKALLOC_DYN : Pseudo<(outs GPR:$rd),
+def PROBED_STACKALLOC_DYN : Pseudo<(outs),
                                (ins GPR:$scratch),
```

Comment:
(nit) not sure if `scratch` is the best name here.
I guess it follows other nodes, but the meaning here is different. Should it be `sp`, maybe?

Reply:
Renamed to $target to match AArch64 and how it's referred to in the custom insertion code where it is called `TargetReg`